.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "mii emu",
+			"type": "cppdbg",
+			"preLaunchTask": "${defaultBuildTask}",
+			"request": "launch",
+			"program": "${workspaceFolder}/build-x86_64-redhat-linux/bin/mii_emu",
```

Comment:
I dunno how to deal with this though. I suspect you'll need to change it to be a static path?

Reply:
Thanks for that BTW -- I haven't merged because I haven't found a nice way to making that patch work for any platform (ie redhat, debian etc) -- perhaps I'll add a symlink to the exec in the top fdirectory at some point...